index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Hussen Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <!-- the toolbar (or header) section -->
+      <header class="topBar-nav">
+        <div class="toolBar-nav">
+          <div class="logo-setting">
+            <div class="logo">
+              <a href="#"><img src="img/My-Logo.png" alt="HY" /></a>
+            </div>
+          </div>
+          <nav class="dash-setting">
+            <div class="icons">
+              <div class="icon-Union">
+                <a href="#"><img src="img/Union.png" alt="Three Dashs" /></a>
+              </div>
+            </div>
+          </nav>
+        </div>
+      </header>
+      <section class="content-section">
+        <div class="bg-shapes">
+          <div class="text-1">
+            <h2>I’m Hussen Glad to see you!</h2>
```

Comment:
Kindly make this an `h1` tag instead of an `h2` tag since it is the main title. This will help for search engine optimization purposes.